### DISCUSSION TITLE
Color change for at risk counts

### DISCUSSION THREAD

**Discussion by gsp97** [CamDavidsonPilon/lifelines#1289] [Q&A]
Hi Cameron,

Thanks for extending your helping hand and building the package . This is making a KM work get completed at ease. 
I am working on KM cumulative density and I have 6-7 groups where I need at risk counts to be represented in the same color as of the lines are being represented in the curve.
I tried using color = list of colors or color = ax which is not a right way
Please do suggest me a solution
![image](https://user-images.githubusercontent.com/83910487/121509872-b036e680-ca04-11eb-96aa-f50b3fa10f1f.png)

Thanks in advance!!

**Comment by CamDavidsonPilon** [comment]
Hi @gsp97, hm interesting. Unfortunately, given how we implemented the table, it's not possible to add colors to the _rows_.

**Reply by gsp97** [reply]
Is there a way that it can be changed in the package code?
Or would it be possible to implement it using external code push

**Reply by CamDavidsonPilon** [reply]
I don't think so. We create the tables by making each column an "xtick label", i.e. the table is constructed like this:
<img width="543" alt="Screen Shot 2021-06-10 at 2 35 16 PM" src="https://user-images.githubusercontent.com/884032/121579039-5381fe80-c9f9-11eb-927d-0b752aec5fe9.png">


While it is possible in matplotlib to change xtick label colors - that would only change the colors of _columns_, but you want rows. 

So without an alternative implementation of at-risk tables, I don't think this can be done.
